File: vpn-edge-agent/tests/client_payload.js
import io from "socket.io-client";

const socket = io("http://10.0.0.1:9000");

socket.on("connect", () => {
	console.log("Conectado");
	socket.emit("data", { mensaje: "Hola servidor" });
});

socket.on("response", (msg) => {
	console.log("Respuesta:", msg);
});

File: vpn-edge-agent/tests/client_payload.py
import socketio

sio = socketio.Client()

@sio.event
def connect():
    print("Conectado")
    sio.emit("data", {"mensaje": "Hola servidor"})

@sio.on("response")
def on_response(msg):
    print("Respuesta:", msg)

@sio.event
def disconnect():
    print("Desconectado")

sio.connect("http://10.0.0.1:9000")
sio.wait()
